modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc

```diff
@@ -1,6 +1,13 @@
 [id="creating-a-telemetry-plugin_{context}"]
 = Creating A Telemetry Plug-in
 
+[WARNING]
+====
+This page explains how to create a back-end telemetry plug-in for {prod-short} workspaces, which will be deprecated in favour of {devworkspace}s.
```

Comment:
We can't write about the future in the docs.
Will removing this first sentence of this warning admonition make a difference to the meaning of the second sentence?